addons/account/models/company.py

```diff
@@ -98,6 +98,9 @@ class ResCompany(models.Model):
 
     invoice_is_email = fields.Boolean('Email by default', default=True)
     invoice_is_print = fields.Boolean('Print by default', default=True)
+    account_credit_limit = fields.Boolean(string='Credit Limit', help='Enables the use of credit limit on Partners.')
+    account_default_credit_limit = fields.Monetary(string='Default Credit Limit',
+        help='This is the default Credit Limit that will be used on Partners that do not have a specific limit on them.')
```

Comment:
Since we are using company dependent fields, why are the default values stored on the company when the default value for company dependent fields is stored in `ir.property` without a `res_id`?

Reply:
I tried removing that field and using `_set_default()` on `res.config.settings` instead of having a related field, and it worked fine for the partners, except that in the settings view, the value remained "0" even after defining a default limit :man_shrugging: